test/server/routes/applications.test.js

```diff
@@ -391,10 +391,10 @@ describe('routes/applications.js', () => {
         expect(stubNotifications).to.have.been.calledOnce;
       });
   });
-  it('should serve up the delete key page on the delete path when got with an error', () => {
+  it('should serve up the delete key page on the delete path when got with an Incorrect Key Type Error', () => {
     const slug = '/manage-applications/mockAppId/mockKeyType/mockKeyId/delete/mockEnv';
     const genericServerException = exceptions.genericServerException;
-    const testErr = new Error('Test error');
+    const testErr = new Error('Incorrect Key Type');
```

Comment:
Could we be more consistent about the error message and bring this in line with Add key?

Reply:
Is `Validation error` what you're referring to?

Reply:
No I mean during the similar if check on when selecting which add key method to use they use 'Could not match Key Type'
I'm happy to go either way, I mostly want them to match, but I have a slight preference for could not match because maybe in the future we add a different key type.

Reply:
Okay you meant in the actual service, I will change it match now.